bazel/ray_deps_setup.bzl

```diff
@@ -92,7 +92,6 @@ def ray_deps_setup():
         sha256 = "76a33e2136f23971ce46c72fd697cd94dc9f73d56ab23b753c3e16854c90ddfd",
         strip_prefix = "protobuf-2c5fa078d8e86e5f4bd34e6f4c9ea9e8d7d4d44a",
         urls = [
-            # https://github.com/protocolbuffers/protobuf/commits/v23.4
```

Comment:
why removing this link?

Reply:
It's not used, I'm against keep unused / commented-out code in our codebase for no reason.

Reply:
This is a useful comment (not code) that tells us the protobuf version we are using.

Reply:
I added another comment for version.